tapchannel/aux_traffic_shaper.go

```diff
@@ -235,15 +246,26 @@ func (s *AuxTrafficShaper) PaymentBandwidth(htlcBlob,
 		// channel _does_ have available bandwidth.
 		return lnwire.NewMSatFromSatoshis(btcutil.MaxSatoshi), nil
 
-	case htlcAssetAmount != 0 && htlcAssetAmount > computedLocal:
+	case htlcAssetAmount > computedLocal:
 		// The asset balance of the channel is simply not enough to
 		// route the asset units, we report 0 bandwidth in order for the
 		// HTLC to fail back.
 		return 0, nil
+
+	default:
```

Comment:
This should not be possible, so it's just confusing to have the `default` case.
In fact, now that we definitely return from this function, we can turn the first `case` into an `if`. Then have the second `case` non-indented as the implicit `else`.

Reply:
It's should return an error if it ever reaches a point it shouldn't.